File: backend/seed/seed_bread_goods.py
import sys
import os

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))

from app import app
from models.bread_good import db, BreadGood

bread_data = [
   {
    "title" : "VINYL",
    "price" : "34.99",
    "image" : '/images/homepage/musicStore/vinyl.png',
    "link" : "/",
   },
   {
    "title" : "CD",
    "price" : "14.99",
    "image" : '/images/homepage/musicStore/cd.png',
    "link" : "/",
   },
   {
    "title" : "CASSETTE",
    "price" : "14.99",
    "image" : 'images/homepage/musicStore/cassette.png',
    "link" : "/",
   },
]

with app.app_context():
    for data in bread_data:
        bread = BreadGood(**data)
        db.session.add(bread)
    db.session.commit()
    print("✅ Sofi Tukker Bread goods seeded served!")

File: prisma/seed-bread-goods.ts
import "dotenv/config";
import prisma from "../lib/prisma";

const breadGoods = [
  {
      title: "VINYL",
      price: "34.99",
      image: '/images/homepage/musicStore/vinyl.png',
      link: "/",
  },
  {
    title: "CD",
    price: "14.99",
    image: '/images/homepage/musicStore/cd.png',
    link: "/",
  },
  {
    title: "CASSETTE",
    price: "14.99",
    image: '/images/homepage/musicStore/cassette.png',
    link: "/",
  },
];

async function main() {
  await prisma.breadGood.deleteMany();

  await prisma.breadGood.createMany({
    data: breadGoods,
  });

  console.log("Seeded bread_goods:", breadGoods.length);
}

main()
  .catch((e) => {
    console.error(e);
    process.exit(1);
  })
  .finally(async () => prisma.$disconnect());
